app/config/routes.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  vetVisits: 'claim/vet-visits'
```

Comment:
should be 'vet-visits' not 'claim/vet-visits'

Reply:
Thanks for your comment. Done!